server/plugin.go

```diff
@@ -53,6 +53,8 @@ const (
 	actionDeleted          = "deleted"
 	reactionSetAction      = "set"
 	reactionUnsetAction    = "unset"
+	subscriptionConnected  = "connected"
```

Comment:
This constant is already defined i think

Reply:
same https://github.com/mattermost/mattermost-plugin-msteams-sync/pull/354#discussion_r1368341106